site-src/index.md

```diff
@@ -1,111 +1,183 @@
 ## What is the Gateway API?
 
-Gateway API is an open source project managed by the [SIG-NETWORK][sig-network]
-community. It is a collection of resources that model service networking 
-in Kubernetes. These resources - `GatewayClass`, `Gateway`, `HTTPRoute`, 
-`TCPRoute`, `Service`, etc - aim to evolve Kubernetes service networking through 
-expressive, extensible, and role-oriented interfaces that are implemented by 
-many vendors and have broad industry support. 
+Gateway API is an open source project managed by the
+[SIG-NETWORK][sig-network] community. It is a collection of resources that
+model service networking in Kubernetes. These resources -
+[`GatewayClass`][GatewayClass], [`Gateway`][Gateway],
+[`HTTPRoute`][HTTPRoute], [`TCPRoute`][TCPRoute], etc., as well as the
+Kubernetes [`Service`][Service] resource - aim to evolve Kubernetes service
+networking through expressive, extensible, and role-oriented interfaces that
+are implemented by many vendors and have broad industry support.
+
+[GatewayClass]: /api-types/gatewayclass
+[Gateway]: /api-types/gateway
+[HTTPRoute]: /api-types/httproute
+[TCPRoute]: /api-types/tcproute
+[Service]: https://kubernetes.io/docs/concepts/services-networking/service/
 
 ![Gateway API Model](./images/api-model.png)
 
+The Gateway API was originally designed to manage traffic from clients outside
+the cluster to services inside the cluster -- the _ingress_ or
+[_north/south_][north/south traffic] case. Over time, interest from service
+mesh users prompted the creation of the [GAMMA initiative][gamma] to define
+how the Gateway API could also be used for inter-service or [_east/west_
+traffic][east/west traffic] within the same cluster.
+
+## Gateway API for Ingress <a name="for-ingress"></a>
+
+When using the Gateway API to manage ingress traffic, the [Gateway] resource
+defines a point of access at which traffic from outside clients can be routed
+into the cluster ([north/south traffic]). (If you're familiar with the older
```

Comment:
Tiny nit: Always need to be very careful to avoid limiting the scope of the API here. I think Gateway API can and should be used to route traffic within a cluster (not necessarily just from outside to inside) and also that it should be used to route traffic to endpoints outside a cluster (ie multi-cluster Services or hybrid deployments that are only half on k8s). I don't actually know how to rephrase this in a way that makes sense but also doesn't exclude those use cases though, so for now just leaving a comment so we don't lose track of it.

Reply:
Does 5783333f78307a4a037b61c111788454659a692b look good for this?